addOns/pscan/gradle.properties

```diff
@@ -0,0 +1,4 @@
+version=0.0.1
+release=false
+zap.maven.publish=true
+zap.maven.pom.inceptionyear=2024
```

Comment:
Should we be adding this on others?

Reply:
This is only needed in add-ons that are expected to be released to Maven Central, this one should be for other add-ons to depend on it in time.